packages/playwright-test/src/reporters/html.ts

```diff
@@ -147,7 +149,7 @@ function standaloneDefaultFolder(): string {
   return reportFolderFromEnv() ?? defaultReportFolder(process.cwd());
 }
 
-export async function showHTMLReport(reportFolder: string | undefined, testId?: string) {
+export async function showHTMLReport(reportFolder: string | undefined, host: string = 'localhost', port: number = 9223, testId?: string) {
```

Comment:
```suggestion
export async function showHTMLReport(reportFolder: string | undefined, host: string = '127.0.0.1', port: number = 9223, testId?: string) {
```

I think we want this instead.